roundware/api2/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
```

Comment:
Good point. Now that I look around, however, it seems that many files are missing these licensing lines. If I make other changes to this pull request, I will certainly address this. For the more general case, see #284